client/src/app/tabs/panel/tabs/LintingTab.less

```diff
@@ -1,52 +1,55 @@
+:root {
+  --linting-issue-background-color: var(--color-grey-225-10-95);
+  --linting-issue-border-color: var(--color-grey-225-10-95);
+
+  --linting-issue-padding: 8px;
```

Comment:
`--linting-issue-padding: 6px 8x;` as agreed on before to reduce the line height